meta-ostro/conf/local.conf.sample

```diff
@@ -35,6 +26,9 @@
 # For Intel Edison:
 #MACHINE ?= "edison"
 #
+# For BeagleBone and BeagleBone Black:
```

Comment:
We don't support original beaglebone (I doubt even if it's available to purchase anymore), only BBB.